docs/patching-pega-deployment.md

```diff
@@ -70,11 +62,12 @@ To complete a zero downtime patch, you must configure the following settings in
 
 - Update the tagging details, including the version and date of your latest Pega-provided `platform/search` Docker image, that you downloaded for your patch.
 
-- In the installer section of the Helm chart, update the following: 
+- In the installer section of the Helm chart, update the following:
   - Update the tagging details, including the version and date of your Pega-provided `platform/installer` Docker image, that you downloaded to support your patch.
   - Specify an `out-of-place` upgrade to apply a patch using the zero-downtime patch process.
```

Comment:
Here as well, regarding the deprecated usage of `out-of-place`

Reply:
what is it supposed to be? is it now zero-downtime?

Reply:
@taz-pega-work yes zero-downtime is the upgradeType which supports patch upgrades. out-of-place is a deprecated name now.